tests/integration/test_integ.py

```diff
@@ -0,0 +1,129 @@
+# HOW TO EXECUTE?
+# First install the package: pip install pytest-docker-compose
+# Run with: pytest --docker-compose="path_to_docker_compose_yml" test_integ.py
+
+import pytest
+import os
+import subprocess
+import unittest
+import time
+import sys
+import logging
+
+
+class TestIntegratedTool(unittest.TestCase):
+
+
+    def setUp(self):
+        print("/************************************* Entered setUp *************************************/")
+
+        self.mounted_files = []
+        self.FUSE_MOUNTED_FILES_LIST = "/tmp/Mounted_files.lst"
+        self.MDH_CORE = "core-sfs"
+        self.MOUNTED_FROM = "/home/sfsuser/Vaidehi/amos-rep/test_tree"
+
+
+        # Check if mdh core is running
+        is_core_up = subprocess.Popen(["docker", "container", "inspect", "-f",
```

Comment:
please user the docker library for accessing the containers: https://docker-py.readthedocs.io/en/stable/containers.html

Reply:
I don't want to change this code again. I already tried working with this library earlier, after which I switched to using subprocess lib to run the docker.